agent/agents/postgres/parser/testdata/query019.json

```diff
@@ -1,3 +1,7 @@
 {
-  "error": "error on parsing sql query: subquery in FROM must have an alias"
+  "tables": [
+    "credit_card_wo_complaints",
+    "without_complaints"
+  ],
+  "error": ""
 }
```

Comment:
query by itself is incorrect but I see no harm in extracting tables from incorrect query